src/components/WeekDayPicker/index.d.ts

```diff
@@ -0,0 +1,22 @@
+import { ReactNode, ChangeEvent } from 'react';
+import { BaseProps } from '../types';
+
+type WeekDays = 'monday' | 'tuesday' | 'wednesday' | 'thursday' | 'friday' | 'saturday' | 'sunday';
+type Value = WeekDays | WeekDays[];
+
+export interface WeekDayPickerProps extends BaseProps {
+    id?: string;
+    name?: string;
+    value?: Value;
+    label?: ReactNode;
+    hideLabel?: boolean;
+    disabled?: boolean;
+    required?: boolean;
+    readOnly?: boolean;
+    multiple?: boolean;
+    error?: ReactNode;
+    onChange?: (value: Value) => void;
```

Comment:
we should update this file with the new API
remove hideLabel
add availableDates and bottomHelpText